src/main/java/org/opentripplanner/street/model/edge/TemporaryPartialStreetEdge.java

```diff
@@ -40,17 +40,6 @@ public boolean isEquivalentTo(Edge e) {
     return (e == this || e == parentEdge);
   }
 
-  @Override
```

Comment:
Since we don't need the override anymore, can you make the base method final?

Reply:
Done.